spark/src/test/scala/org/apache/comet/CometExpressionCoverageSuite.scala

```diff
@@ -284,7 +324,7 @@ class CometExpressionCoverageSuite extends CometTestBase with AdaptiveSparkPlanH
                   datafusionMessage = dfMessage.getOrElse("")),
                 group = func.group)
 
-            case e =>
```

Comment:
This catches all Throwables. If this is really intended, use `case e : Throwable` to clear this warning.